models/accounts.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	"database/sql"
+	"time"
+)
+
+type Accounts struct {
+	Account        int       `json:"account_id"`
+	AccountNum     string    `json:"account_num"`
+	AccountBalance int       `json:"account_balance"`
+	CreatedAt      time.Time `json:"created_at"`
+}
+
+func GetBalanceAcc(accNum string, db *sql.DB) (acc Accounts, err error) {
+	var balanceFloat float32
+	err = db.QueryRow("SELECT account_balance from accounts where account_num = ($1) ", accNum).Scan(&balanceFloat)
```

Comment:
add error checking here so if there's an error when running db query, the next line will not run